File: src/main/java/org/shark/example/datasource/cassandra/pojo/AccountDo.java
package org.shark.example.datasource.cassandra.pojo;

import lombok.Data;
import org.springframework.data.cassandra.core.mapping.CassandraType;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

import java.util.UUID;

@Data
@Table("account")
public class AccountDo {

    @CassandraType(type = CassandraType.Name.UUID)
    @PrimaryKey
    private UUID id;

    private String account;

    private String password;
}

File: src/main/kotlin/org/shark/example/datasource/cassandra/pojo/AccountDo.kt
package org.shark.example.datasource.cassandra.pojo

import org.springframework.data.cassandra.core.mapping.CassandraType
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table
import java.util.UUID

@Table("account")
data class AccountDo(
    @CassandraType(type = CassandraType.Name.UUID)
    @PrimaryKey
    var id: UUID? = null,
    var account: String? = null,
    var password: String? = null
)
